conf/modules.config

```diff
@@ -277,7 +277,7 @@ process {
     }
 
     withName: KRAKEN2_KRAKEN2 {
-        ext.args = { "${meta.db_params}" }
+        ext.args = params.kraken2_save_minimizers ? { "${meta.db_params} --report-minimizer-data" } : { "${meta.db_params}" }
```

Comment:
Does this result in a new file? Or just adds an addition column to the output for example?

Reply:
Just adds two more columns to the report for a total of 8 columns.